src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -3,13 +3,15 @@
     Choose a base template
 
   %fieldset.templateslist
+    - first_template = nil
     - @projects.each do |project|
       %dl.templateslist
         %dt
           = link_to(title_or_name(project), project_show_path(project))
-        - project.packages.each do |template|
+        - project.packages.each.with_index(1) do |template, index|
```

Comment:
@bgeuken good catch! I will change it in #2449 then because we change the loading anyway for remote image templates!